src/pr-info.ts

```diff
@@ -498,9 +502,11 @@ function getDangerLevel(categorizedFiles: readonly FileInfo[]): DangerLevel {
         return "Infrastructure";
     }
     const packagesTouched = getPackagesTouched(categorizedFiles);
+    const sourcePackagesTouched = getPackagesWithSourceFilesTouched(categorizedFiles);
+
     if (packagesTouched.length === 0) {
         return "Infrastructure";
-    } else if (packagesTouched.length > 1) {
+    } else if (sourcePackagesTouched.length > 1) {
```

Comment:
I completely agree that it would be best to do something better with the multiple packages case, but I worry that this makes it easy to miss things, since it will show the multiple label only sometimes.

What I was thinking of is separating things so that multiple-packages is one attribute, and config/no-tests would be another--?  And if the search is done for all files then (for example) you won't get the untested label if you edited a definition in one package and a test in another (but you will get the multiple packages of course).

But regardless, it looks like this change wouldn't affect that PR (DefinitelyTyped/DefinitelyTyped#46321) since it has changes to two definitions files in two packages..